cfpb/cfgov-refresh
# Moving owning-a-home/closure-disclosure pages into cfgov-refresh

## Changes

- Modified CSS, HTML, and images to move `owning-a-home/closing-disclosure/` into cfgov-refresh. 

## Testing

1. Run `gulp build`.
2. Visit `https://www.consumerfinance.gov/owning-a-home/closing-disclosure/` and 
`http://localhost:8000/owning-a-home/closing-disclosure/`.
3. Click the expandables on and ensure they expand / contract as expected.
4. Compare localhost to production.
5. Mouse over the highlighted areas on the image and ensure the related expendables receive focus.
6. Click the 'Get definitions` tab and ensure the expandable terms switch, as expected.
7. Click the pagination buttons and ensure the pages fadeIn/Out, as expected.
8. Resize the browsers and ensure the form explainer images resize properly.
9. Scroll the pages and ensure the form explainer images doesn't overlap the ratings well.

## Screenshots


## Notes

- The `focus-target.js` code runs on all pages, and adds an event listener to the all anchor links. This causes the page to jump unexpectedly in some situations via Chrome.  We should modify the modules to require the code instead of including it in common.js.

## Todos

- Write unit tests.

## Checklist
